_raml/apps/products/examples/get_product_productid_crossselling_response.json

```diff
@@ -160,6 +160,7 @@
         "sfUrl": "http://pm.epages.com/epages/apidocu.sf/?ObjectPath=/Shops/apidocu/Products/Cust001",
         "productNumber": "Cust001",
         "isProductNumberVisible": true,
+        "isNew": false,
```

Comment:
I'm not sure but I think this line needs to be added once more in this file after line 283.